### DISCUSSION TITLE
How to move songs, drumkits and patterns etc to a new computer

### DISCUSSION THREAD

**Discussion by TonyTheFerg** [hydrogen-music/hydrogen#1963] [Hydrogen on Windows]
Hi all, I recently switched to a new Windows 11 computer. I previously had version 0.9.7 installed on Windows 10. 

I copied my files (kits, patterns, songs) from `C:\Users\user\.hydrogen\data`, but the drumkit I had used (DeathMetal.h2drumkit) for each project is not properly recognized:

I am seeing this: 

> "some samples in this song could not be loaded. This may be because the use an older default drumkit. This might be fixed by opening a new drumkit"

Loading the migrated drumkit causes the instruments to be loaded in a different order in the patterns, which then causes the patterns to be assigned to the wrong instruments. I didn't copy any other data from other hydrogen directories. I am hoping there is a file I am missing so that I don't need to reconstruct all these patterns?

Thanks very much for your help, and let me know if I need to clarify anything!

**Comment by theGreatWhiteShark** [comment]
Hey @TonyTheFerg ,

Could you attach the [log messages](https://github.com/hydrogen-music/hydrogen/wiki/Accessing-log-messages-of-Hydrogen#windows) Hydrogen prints during startup?

> Loading the migrated drumkit causes the instruments to be loaded in a different order

Hmm. This does not sound nice.

Hydrogen does indeed tries to update a drumkit's definition file (each drumkit consists of a bunch of samples and a `drumkit.xml` gluing every together). But it should keep a backup (using the suffix `.bak`) of the original file. So, nothing should be lost. Could you upload both `C:\Users\user\.hydrogen\data\drumkits\DeathMetal\drumkit.xml` and `C:\Users\user\.hydrogen\data\drumkits\DeathMetal\drumkit.xml.***.bak` so I can have a look?
